README.md

```diff
@@ -1,2 +1,121 @@
 # vm-device
-A virtual machine device model crate
+
+The `vm-device` crate provides:
+* device traits defining read and write operations on specialized buses
```

Comment:
Hmm I think I am missing an introduction about what a bus is in the context of a VMM (we are starting this too abruptly I guess :-?). Maybe we can enumerate what is needed in order to have a functional device attached to a VMM, e.g. a bus, a backend etc., something similar to the enumeration [here](https://github.com/rust-vmm/vm-superio/blob/main/README.md#design).